novu/api/environment.py

```diff
@@ -67,3 +67,15 @@ def api_keys(self) -> Iterator[EnvironmentApiKeyDto]:
         results = self.handle_request("GET", f"{self._environment_url}/api-keys")["data"]
         for result in results:
             yield EnvironmentApiKeyDto.from_camel_case(result)
+
+    def regenerate_api_key(self) -> Iterator[EnvironmentApiKeyDto]:
+        """Regenerate an environment api key
+
+        Yields:
+            Mapped environment's api key
+        """
+        results = self.handle_request("POST", f"{self._environment_url}/api-keys/regenerate")["data"]
+        if not isinstance(results, list):
+            results = [results]
+        for result in results:
+            yield EnvironmentApiKeyDto.from_camel_case(result)
```

Comment:
API docs say that we only retrieve list from them, we should not add this I think

```suggestion
        for result in results:
            yield EnvironmentApiKeyDto.from_camel_case(result)
```